src/utils/converters.ts

```diff
@@ -0,0 +1,17 @@
+import BigNumber from "bignumber.js";
+
+const MASSA_SCALING_FACTOR = 9;
+
+export const toMAS = (inputAmount: number | string): number => {
+    const amount = new BigNumber(inputAmount);
+    const scaleFactor = (new BigNumber(10)).pow(new BigNumber(MASSA_SCALING_FACTOR));
+    const amountScaled = amount.times(scaleFactor);
+    return parseInt(amountScaled.toFixed(0));
+};
+
+export const fromMAS = (inputAmount: number | string): BigNumber => {
+    const amount = new BigNumber(inputAmount);
+    const scaleFactor = (new BigNumber(10)).pow(new BigNumber(MASSA_SCALING_FACTOR));
+    const amountScaled = amount.dividedBy(scaleFactor);
+    return amountScaled;
```

Comment:
maybe something like this is more efficient ?
   ```
    const amount = new BigNumber(inputAmount).toFixed(0);
    return BigNumber(amount.substring(0, amount.length - MASSA_SCALING_FACTOR));

Reply:
I have fixed that part now.